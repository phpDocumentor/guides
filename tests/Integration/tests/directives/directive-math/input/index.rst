Directive tests
===============

..  math::

    <mrow>
        <mi>α</mi>
        <mi>t</mi>
        <mo stretchy="false">(</mo>
        <mi>i</mi>
        <mo stretchy="false">)</mo>
        <mo>=</mo>
        <mi>P</mi>
        <mo stretchy="false">(</mo>
        <mi>O</mi>
        <msub>
            <mi>t</mi>
            <mo>,</mo>
            <mi>i</mi>
        </msub>
        <mo>=</mo>
        <mi>S</mi>
        <mi>i</mi>
        <mi>λ</mi>
        <mo stretchy="false">)</mo>
    </mrow>
