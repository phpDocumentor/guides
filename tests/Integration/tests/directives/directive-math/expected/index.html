<!-- content start -->
    <div class="section" id="directive-tests">
            <h1>Directive tests</h1>
            <math><mrow>
    <mi>α</mi>
    <mi>t</mi>
    <mo stretchy="false">(</mo>
    <mi>i</mi>
    <mo stretchy="false">)</mo>
    <mo>=</mo>
    <mi>P</mi>
    <mo stretchy="false">(</mo>
    <mi>O</mi>
    <msub>
        <mi>t</mi>
        <mo>,</mo>
        <mi>i</mi>
    </msub>
    <mo>=</mo>
    <mi>S</mi>
    <mi>i</mi>
    <mi>λ</mi>
    <mo stretchy="false">)</mo>
</mrow></math>
    </div>
<!-- content end -->
